tools/targets/NCS.py

```diff
@@ -48,17 +51,41 @@ def add_fib_at_start(arginput):
     regions = len(start_end_pairs)
 
     if regions == 1:
+        #single range indicating fits within first flash block (<320K)
         start, end = start_end_pairs[0]
+        print("Memory start 0x%08X, end 0x%08X" % (start, end))
```

Comment:
Previously accepted versions of this file have used print statements by default.  Is that not allowed?  I'm fine removing these print statements if needed.